Manito/Manito/Screens/Interaction/View/OpenManittoView.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  OpenManittoView.swift
+//  Manito
+//
+//  Created by SHIN YOON AH on 2023/03/27.
+//
+
+import UIKit
+
+import SnapKit
+
+protocol OpenManittoViewDelegate: AnyObject {
+    func confirmButtonTapped()
+}
+
+final class OpenManittoView: UIView {
+
+    private enum InternalSize {
+        static let collectionHorizontalSpacing: CGFloat = 29.0
+        static let collectionVerticalSpacing: CGFloat = 37.0
+        static let cellLineSpacing: CGFloat = 20.0
+        static let cellInteritemSpacing: CGFloat = 39.0
+        static let cellItemSize: CGFloat = floor((UIScreen.main.bounds.size.width - (collectionHorizontalSpacing * 2 + cellInteritemSpacing * 2)) / 3)
+        static let collectionSectionInset = UIEdgeInsets(top: collectionVerticalSpacing,
+                                                         left: collectionHorizontalSpacing,
+                                                         bottom: collectionVerticalSpacing,
+                                                         right: collectionHorizontalSpacing)
+    }
+
+    // MARK: - ui component
+
+    private let collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        flowLayout.sectionInset = InternalSize.collectionSectionInset
+        flowLayout.minimumLineSpacing = InternalSize.cellLineSpacing
+        flowLayout.minimumInteritemSpacing = InternalSize.cellInteritemSpacing
+        flowLayout.sectionHeadersPinToVisibleBounds = true
+        flowLayout.itemSize = CGSize(width: InternalSize.cellItemSize, height: InternalSize.cellItemSize)
+        return flowLayout
+    }()
+    private lazy var manittoCollectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: self.collectionViewFlowLayout)
+        collectionView.backgroundColor = .clear
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.isScrollEnabled = false
+        collectionView.register(cell: OpenManittoCollectionViewCell.self,
+                                forCellWithReuseIdentifier: OpenManittoCollectionViewCell.className)
+        return collectionView
+    }()
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .font(.regular, ofSize: 34)
+        label.text = TextLiteral.openManittoViewControllerTitle
+        return label
+    }()
+    private let popupView: OpenManittoPopupView = OpenManittoPopupView()
+
+    // MARK: - property
+
+    private let totalCount = 10.0
```

Comment:
p1;

타입 명시적으로 변경 부탁드립니다! 
```suggestion
    private let totalCount: Double = 10.0
```

Reply:
[01b365f](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/433/commits/01b365f67a7f7470ad78c5a1722258463b9cf5eb)

타입 명시했습니다!